lib/sensors/include/FSR.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "mbed.h"
+
+namespace force_sensor {
+
+class FSR {
+ public:
+  // rule-of-5
+  FSR() = delete;
+  explicit FSR(const PinName read_pin, const int sensitivity_resistor = DEFAULT_RESISTOR,
+               const int VCC = MAX_VOLTAGE_MILLI_VOLTS);
+  FSR(const FSR& copy)            = delete;
+  explicit FSR(const FSR&& mover) = delete;
+  FSR& operator                   =(const FSR& copy);
+  FSR& operator=(FSR&& mover) = delete;
+  ~FSR()                      = default;
+
+  // @param - force:
+  //          gets filled with the force measurements in Newtons - pass by reference
+  // @return - 0 for successful read and -1 on failed read
+  int getForce(float& force);
+
+ private:
+  // special constants needed for calculation
+  constexpr static int LARGE_FORCE{1000};  // ooh magic number
```

Comment:
maybe don't write "magic number" and stuff as its not very helpful. If you got it from a datasheet could you add a link to it? Otherwise maybe some explanation on why the value is as it is would be helpful